skglm/prototype_PN/L1_penalty.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
```

Comment:
you can directly edit the `L1` class. It will better highlight which changes are needed, and reduce the amount of duplicated us for reviewers to read